src/elm/Page/Community/ActionEditor.elm

```diff
@@ -774,12 +774,19 @@ viewSelectedVerifiers shared model =
                 |> List.map
                     (\p ->
                         div
-                            [ onClick (OnRemoveVerifier p)
-                            , class "flex flex-col m-2 items-center"
-                            ]
-                            [ Avatar.view ipfsUrl p.avatar "h-7 w-7"
-                            , span [ class "mt-1 text-red font-sans text-caption uppercase leading-caption" ]
-                                [ text_ "community.actions.form.remove_verifier" ]
+                            [ class "flex flex-col m-3 items-center" ]
+                            [ div [ class "relative h-10 w-12 ml-2" ]
+                                [ Avatar.view ipfsUrl p.avatar "h-10 w-10"
+                                , div
+                                    [ onClick (OnRemoveVerifier p)
+                                    , class "absolute top-0 right-0 z-10 bg-red rounded-full h-6 w-6 flex items-center"
+                                    ]
+                                    [ span [ class "text-white text-body font-normal font-sans p-2" ]
+                                        [ text "x" ]
```

Comment:
Hey @zacck please use the asset instead of a text. Link on Figma: https://www.figma.com/file/H5yiyaDbYmNwYrOBnPBgss/CreateAction?node-id=227%3A7

Reply:
@lucca65  done